discovery/swarmkit/swarmkit.go

```diff
@@ -142,7 +143,18 @@ func (d *Discovery) Watch(stopCh <-chan struct{}) (<-chan discovery.Entries, <-c
 				}
 				currentEntries = newEntries
 			case err := <-eventsErrChan:
-				errCh <- err
+				log.Debugf("SwarmKit discovery: events stream failed with error: %s", err.Error())
```

Comment:
Do you think this should be logged at a WARN level instead? It should be pretty rare, right?

Reply:
Yeah it should be rare (most commonly will happen if the node itself goes down). If that happens (and once we have leader election), consequences of node failure will be handled in a different place.